rvs/src/rvsmodule.cpp

```diff
@@ -165,26 +165,35 @@ rvs::module* rvs::module::find_create_module(const char* name) {
       libpath += "/";
     } else {
       rvs::options::has_option("pwd", &libpath); // has ending forward slash too
-      libpath += "../lib/rvs/";
+      libpath += "../lib64/rvs/";
```

Comment:
We should change this to use CMAKE_INSTALL_LIBDIR. So that we may not need 2 path check, might need only 1 check here.

Reply:
Looks like the default value setting for CMAKE_INSTALL_LIBDIR is missing in rvs, that seems to have caused the move from lib to lib64. We should add that first and change here also.

Reply:
@nunnikri Ok. We can make that change. But this implementation will work for lib, lib64 and current folder so we will not have to add anything here unless we have a new path of libs.

Reply:
Since we moved to GNU_INSTALL_DIR, it will change based on distros. Major ones are as below.
Ubuntu : lib
RHEL/CENTOS/SLES: lib64
DEBIAN: lib/<multiarch-tuple>

Other distros it might be even different. Since we are not building in Debian in our CI, we dont get other folder. Yes this will work for now to resolve the ticket.  But its better to use CMAKE_INSTALL_LIBDIR, instead of adding cases for each and every cases.

Reply:
Yes, CMAKE_INSTALL_LIBDIR have to be done for lib path standardization across currently supported distros.